easybuild/easyconfigs/r/R/R-3.1.3-goolf-1.7.20-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,386 @@
+    ('nlme', '3.1-120', ext_options),
+    ('mgcv', '1.8-5', ext_options),
+    ('minqa', '1.2.4', ext_options),
+    ('nloptr', '1.0.4', ext_options),
+    ('RcppEigen', '0.3.2.4.0', ext_options),
+    ('lme4', '1.1-7', ext_options),
+    ('pbkrtest', '0.4-2', ext_options),
+    ('quantreg', '5.11', ext_options),
+    ('car', '2.0-25', ext_options),
+    ('colorspace', '1.2-6', ext_options),
+    ('robustbase', '0.92-3', ext_options),
+    ('sp', '1.0-17', ext_options),
+    ('vcd', '1.3-2', ext_options),
+    ('snowfall', '1.84-6', ext_options),
+    ('rpart', '4.1-9', ext_options),
+    ('mice', '2.22', ext_options),
+    ('urca', '1.2-8', ext_options),
+    ('fracdiff', '1.4-2', ext_options),
+    ('logistf', '1.21', ext_options),
+    ('akima', '0.5-11', ext_options),
+    ('bitops', '1.0-6', ext_options),
+    ('boot', '1.3-15', ext_options),
+    ('mixtools', '1.0.2', ext_options),
+    ('cluster', '2.0.1', ext_options),
+    ('gclus', '1.3.1', ext_options),
+    ('coda', '0.17-1', ext_options),
+    ('codetools', '0.2-11', ext_options),
+    ('foreach', '1.4.2', ext_options),
+    ('doMC', '1.3.3', ext_options),
+    ('DBI', '0.3.1', ext_options),
+    ('foreign', '0.8-63', ext_options),
+    ('survival', '2.38-1', ext_options),
+    ('gam', '1.09.1', ext_options),
+    ('gamlss.data', '4.2-7', ext_options),
+    ('gamlss.dist', '4.3-4', ext_options),
+    ('hwriter', '1.3.2', ext_options),
+    ('KernSmooth', '2.23-14', ext_options),
+    ('zoo', '1.7-12', ext_options),
+    ('xts', '0.9-7', ext_options),
+    ('TTR', '0.22-0', ext_options),
+    ('quantmod', '0.4-4', ext_options),
+    ('lmtest', '0.9-33', ext_options),
+    ('mnormt', '1.5-1', ext_options),
+    ('mvtnorm', '1.0-2', ext_options),
+    ('pcaPP', '1.9-60', ext_options),
+    ('numDeriv', '2012.9-1', ext_options),
+    ('lava', '1.4.0', ext_options),
+    ('prodlim', '1.5.1', ext_options),
+    ('pscl', '1.4.8', ext_options),
+    ('RSQLite', '1.0.0', ext_options),
+    ('BatchJobs', '1.6', ext_options),
+    ('sandwich', '2.3-2', ext_options),
+    ('sfsmisc', '1.0-27', ext_options),
+    ('spatial', '7.3-9', ext_options),
+    ('VGAM', '0.9-7', ext_options),
+    ('waveslim', '1.7.5', ext_options),
+    ('xtable', '1.7-4', ext_options),
+    ('profileModel', '0.5-9', ext_options),
+    ('brglm', '0.5-9', ext_options),
+    ('deSolve', '1.11', ext_options),
+    ('tseriesChaos', '0.1-13', ext_options),
+    ('tseries', '0.10-34', ext_options),
+    ('fastICA', '1.2-0', ext_options),
+    ('R.methodsS3', '1.7.0', ext_options),
+    ('R.oo', '1.19.0', ext_options),
+    ('cgdsr', '1.1.33', ext_options),
+    ('R.utils', '2.0.0', ext_options),
+    ('R.matlab', '3.2.0', ext_options),
+    ('BiocGenerics', '0.12.1', bioconductor_options),
+    ('Biobase', '2.26.0', bioconductor_options),
+    ('S4Vectors', '0.4.0', bioconductor_options),
+    ('IRanges', '2.0.1', bioconductor_options),
+    ('GenomeInfoDb', '1.4.0', bioconductor_options),
```

Comment:
1.2.5 is still availabe via http://bioconductor.org/packages/3.0/bioc/html/GenomeInfoDb.html, which is just a minor change to the 1.2.4 included in existing R 3.1.3 easyconfigs, so use that?

We can sync up the existing R easyconfigs.